addons/crm/models/crm_team.py

```diff
@@ -1,20 +1,52 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models, _
 from odoo.tools.safe_eval import safe_eval
+from odoo.exceptions import ValidationError
 
 
 class Team(models.Model):
     _name = 'crm.team'
     _inherit = ['mail.alias.mixin', 'crm.team']
 
     resource_calendar_id = fields.Many2one('resource.calendar', string="Working Time", help="Used to compute open days")
-    use_leads = fields.Boolean('Leads',
-        help="The first contact you get with a potential customer is a lead you qualify before converting it into a real business opportunity. Check this box to manage leads in this sales team.")
-    use_opportunities = fields.Boolean('Opportunities', default=True, help="Check this box to manage opportunities in this sales team.")
-    alias_id = fields.Many2one('mail.alias', string='Alias', ondelete="restrict", required=True, help="The email address associated with this team. New emails received will automatically create new leads assigned to the team.")
+    use_leads = fields.Boolean(
+        'Leads', default=lambda self: self.user_has_groups('crm.group_use_lead'),
+        help="The first contact you get with a potential customer is a lead you qualify before converting it into a real business opportunity. Check this box to assign leads to this sales channel.")
+    use_opportunities = fields.Boolean('Pipeline', default=True, help="Check this box to manage opportunities in this sales channel.")
+    alias_id = fields.Many2one('mail.alias', string='Alias', ondelete="restrict", required=True, help="The email address associated with this channel. New emails received will automatically create new leads assigned to the channel.")
+    unassigned_leads = fields.Integer(
```

Comment:
Hmm, just realized the same field is also declared in website_crm_score/models/sales_team.py

I'll rename everything :p